### PR TITLE
readme: Running instructions

### LINKED ISSUE #231: readme: add instructions to build and run using kindling

### PR DESCRIPTION
**Pull request by airidaceae**
closes #231
depends on #297

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,6 +142,26 @@ Example:
...
 ```
 
+## Running with Kindling
+The provided Hearth binaries act as a runtime and will do nothing on their own.
+For Hearth to work properly you must give it a WebAssembly script to serve as
+the init system and a directory to serve as its root. Hearth provides a set of
+basic utilities for use with the runtime. These utilities are called Kindling
+and can be found in the [corresponding directory](./kindling). Kindling's root
+and init system can be built with the script found in the tools directory. A
+cargo alias is provided for ease of use.
```
Comment (marceline-cramer) [inline README.md:152]:
```suggestion
and init system can be built with the script found in the tools directory.

A Cargo alias is provided for ease of use:
```

File: README.md [hunk 2]
```diff
@@ -142,6 +142,26 @@ Example:
...
+
+```sh
+cargo build-root # build Kindling using the build script.
+```
+
+Now that the root has been built, Hearth can be ran with kindling using the
+following command in the root of the repository.
```
Comment (marceline-cramer) [inline README.md:159]:
```suggestion
Now that the root has been built, Hearth can be ran with Kindling as its root:
```

File: README.md [hunk 3]
```diff
@@ -142,6 +142,26 @@ Example:
 hearth-server --help # prints usage info for the Hearth server
 ```
 
+## Running with Kindling
+The provided Hearth binaries act as a runtime and will do nothing on their own.
+For Hearth to work properly you must give it a WebAssembly script to serve as
+the init system and a directory to serve as its root. Hearth provides a set of
+basic utilities for use with the runtime. These utilities are called Kindling
+and can be found in the [corresponding directory](./kindling). Kindling's root
```
Comment (marceline-cramer) [inline README.md:150]: Adapt how you want to fit the formatting you'd like.

```suggestion
the init system and a directory to serve as its root. Hearth provides a set of
batteries-included utilities for use with the runtime, named Kindling.
Kindling's source code is located within this repository [here](./kindling). Kindling's root
```

**Review by marceline-cramer — CHANGES_REQUESTED** [review]
minor feedback.

File: README.md (hunk 2 above)
Comment (airidaceae) [reply README.md:160]: This was meant to state that this command only works when in the root of the repository.

File: README.md (hunk 3 above)
Comment (airidaceae) [reply README.md]: making some bigger changes to this paragraph. Let me know what you think.

File: README.md (hunk 2 above)
Comment (marceline-cramer) [reply README.md:162]: The runtime's filesystem root vs the repository root is ambiguous, could you suggest a better way of conveying that?

Comment (airidaceae) [reply README.md:162]: Honestly I think that my original clarification wasnt actually all that necessary.

**Review by marceline-cramer — APPROVED** [review]

**Pull request merged**